app/views/manager/books/index.html.erb

```diff
@@ -32,9 +23,18 @@
       <td><%= book.is_pdf %></td>
       <td><%= book.quantity %></td>
       <td><%= book.price %></td>
+      <td>
+        <div class="center">
+          <%= link_to edit_manager_book_path(book.id) do %>
+            <i class="fa fa-pencil" aria-hidden="true"></i>
+          <% end %>
+          <%= link_to manager_book_path(book.id), method: :delete do %>
+            <i class="fa fa-times" aria-hidden="true"></i>
+          <% end %>
+        </div>
+      </td>
```

Comment:
I think this part is made as a partial that you can use in users, did you check it?

Reply:
do you mean this partial 
![image](https://user-images.githubusercontent.com/29805061/87839376-e0c8db80-c89a-11ea-8f98-e11c28cfc670.png)

you use edit_manager_user_path & manager_user_path. it isn't changeable

Reply:
update it with ruby's send method and use it